services/web/server/src/simcore_service_webserver/users_handlers.py

```diff
@@ -29,12 +38,11 @@ async def get_my_profile(request: web.Request):
 @permission_required("user.profile.update")
 async def update_my_profile(request: web.Request):
     uid = request[RQT_USERID_KEY]
-
-    # TODO: validate
     body = await request.json()
+    updates = ProfileUpdate.parse_obj(body)
```

Comment:
minor: why not using the parse_body_as method? it works beautifully.

Reply:
``parse_body_as`` will create a tmp model and include the model class as ``__root__`` and the it calls ``model.parse_obj``. That is handy when you have classes not derived from ``BaseModel``. If your class is derived from ``BaseModel`` calling the free function is a unnecessary overhead.

If you do not understand what I mean, just check the implementation
https://github.com/pydantic/pydantic/blob/main/pydantic/tools.py#L38